src/components/utils/include/utils/qdb_wrapper/sql/sql_database.h

```diff
@@ -42,12 +42,13 @@ namespace utils {
 namespace dbms {
 
 class SQLQuery;
-
+enum class Persistent {IN_MEMORY};
 /**
  * Represents a connection to a database.
  */
 class SQLDatabase {
  public:
+  SQLDatabase(Persistent);
```

Comment:
@BSolonenko `explicit`?

Reply:
@AKalinich-Luxoft fixed in https://github.com/Ford-Luxoft/sdl_core/pull/17/commits/bd843e40d22a303560a02c26487ca620a7f8dfa5